sdk-lib/src/main/java/cash/z/ecc/android/sdk/model/Received.kt

```diff
@@ -0,0 +1,15 @@
+package cash.z.ecc.android.sdk.model
+
+data class Received(
```

Comment:
Shouldn't these be called ReceivedNote and SentNote? They could be mistakenly thought as transactions

Reply:
I will rename them 👍

Reply:
Actually we went backwards on this and made them more clearly Transactions and not notes.